### PR TITLE
Init module with target in .gitignore

### PR DESCRIPTION
**Pull request by pascalgrimaud**
- fix https://github.com/jhipster/jhipster-lite/issues/10873

### CODE REVIEW

File: src/test/resources/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactoryTest.shouldBuildModuleWithStyle.eslint.config.js.approved.txt [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default typescript.config(
     ignores: ['target/'],
   },
   js.configs.recommended,
+  ...typescript.configs.recommended.map(config => (config.name === 'typescript-eslint/base' ? config : { ...config, files: ['**/*.ts'] })),
```
Comment (DamnClin) [inline src/test/resources/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactoryTest.shouldBuildModuleWithStyle.eslint.config.js.approved.txt:18]: This is really wanted here?

**Review by DamnClin — APPROVED** [review]

File: src/test/resources/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactoryTest.shouldBuildModuleWithStyle.eslint.config.js.approved.txt (hunk 1 above)
Comment (murdos) [reply src/test/resources/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactoryTest.shouldBuildModuleWithStyle.eslint.config.js.approved.txt:18]: No, I'll push another fix :)

Comment (murdos) [reply src/test/resources/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactoryTest.shouldBuildModuleWithStyle.eslint.config.js.approved.txt:18]: I was not able to push to your branch, so I've open a new PR: https://github.com/jhipster/jhipster-lite/pull/10924

Comment (pascalgrimaud) [reply src/test/resources/tech/jhipster/lite/generator/client/react/core/domain/ReactCoreModulesFactoryTest.shouldBuildModuleWithStyle.eslint.config.js.approved.txt:18]: Strange lol

**Pull request closed without merging**